packages/graphql/src/schema/resolvers/wrapper.ts

```diff
@@ -105,10 +118,45 @@ export const wrapResolver =
             executorConstructorParam.bookmarks = context.driverConfig?.bookmarks;
         }
 
+        if (!neo4jDatabaseInfo?.version) {
```

Comment:
Surely this is going to _always_ be undefined, because it is only defined on line 56. I believe this database version fetching will happen on every execution with this current implementation.

Reply:
Misunderstanding here!!